### ISSUE TITLE
BeneshBasics10Review

### ISSUE THREAD

**Issue by rsryman** [rryman/BeneshForBalletBook4#14]
**DONE**

I am beginning to do the BB10 Review, based on "D341.3 Assignments" re Lesson 18. The only relevant questions at this point are #1-4. The real challenge is to fit the widget format. If you think the following ideas will work, I will select/create the appropriate locations and/or directions:

Question 1 based on p. 7: The graphic is an "empty" stage plan with only the column lines (and 9 "hot spots" below, for each graphic to be dragged). There are nine buttons, one for each stage plan sign but in random order. Instruction: 
"Drag each location sign below the corresponding column on the stage plan."

Question 2 based on pp. 8-11: To test location without facing? The graphic is an "empty" stage plan with column lines  and 5-9 "X's" on various locations (and "hot spots" below, for each graphic to be dragged). There are 5-9 buttons, one for each corresponding location sign but in random order. Instruction: 
"Drag each location sign below the corresponding location on the stage plan."

Question 3 based on pp. 8-11: To test location and facing? The graphic is an "empty" stage plan with column lines  and 5-9 Jenna dancers in 1st arabesque in various alignments/facings on various locations (and "hot spots" below, for each graphic to be dragged). There are 5-9 buttons, one for each corresponding direction and location sign but in random order. Instruction: 
"Drag each direction and location sign below the corresponding dancer (Jenna in 1st arabesque) on the stage plan." (We might want more than one page of this format, perhaps with Ben in a simple pose.)

**Comment by rsryman** [comment]
See uploaded folder "BB10_Review" 

I decided to do the DanceForms top view images for 1st arabesque facing the 8 directions, some of these to be superimposed on a stage plan for Question 3a. Please advise if you think we need to same for Ben, e.g., with arms demi-bras: Question 3b. "Drag each direction and location sign below the corresponding dancer (Ben with arms opened forward) on the stage plan."

**Comment by rryman** [comment]
I have uploaded the most recent version of Book 4 to dropbox with the Review of Location Signs completed (see pages 17 and 18). I added another question at the end dealing with identification signs. Let me know if you think this works. I can add more questions if you want with Ben, but personally I don't think we need them. Another question like Q 7 of 7 might be helpful as these are taken directly from page 14. It might be good to quiz them on a different set they can't refer back too. However, this is LOTS of work so I am only willing to do it if you feel it would be very beneficial.
